google-cloud-pubsub/samples/acceptance/subscriptions_test.rb

```diff
@@ -104,12 +107,15 @@
     assert_output "Subscription #{@subscription.name} deleted.\n" do
       delete_subscription subscription_id: @subscription.name
     end
-    @subscription = @topic.subscription @subscription.name
-    refute @subscription
+    assert_raises Google::Cloud::NotFoundError do
+      @subscription = subscription_admin.get_subscription subscription: @subscription.name
```

Comment:
Don't assign to `@subscription` here. It's supposed to raise an error before it gets to that point anyway, so it's confusing to see the assignment.